### ISSUE TITLE
Button Type

### ISSUE THREAD

**Issue by gulgulia17** [profabric/profabric-components-issues#1]
Thanks for this amazing package, I'm currently using it with angular and struggling with defining the button type using this, is there any workaround for this?

**Comment by erdkse** [comment]
hey @gulgulia17, thanks for using this library, I can add button type property, in a short time period I am planning to add more components so it is not a problem.

Can you describe your use case, so I can release a new version with your need.

**Comment by gulgulia17** [comment]
Hey @erdkse,

I'm having trouble understanding how to customize the layout of the button. Currently, I'm trying to override the default border radius, but it's not working, and I couldn't find any relevant information in the documentation. Can you provide guidance on this?

**Comment by erdkse** [comment]
Of course! @gulgulia17, Documentation is still under progress. I am planning to release ~10 more components soon and I will add documentation with them. These are scoped components with shadow dom. You need to specify custom css rules such as `--pf-color`, `--pf-width`. The users will eventually be able to create their own Design System easily.

But unfortunately because of I am the only developer, these things take some time. Sorry for it.

I will inform you here.

**Comment by erdkse** [comment]
Please send me a mail that covers all of your needs and I will publish a beta release with them @gulgulia17
